src/components/ResizeDetector/ResizeDetector.tsx

```diff
@@ -2,8 +2,16 @@ import * as React from 'react';
 import { PURE } from 'dx-util/src/react/pure';
 import { themr } from 'react-css-themr';
 import { ComponentClass } from 'react';
-
-import * as detectorFactory from 'element-resize-detector';
+let detectorFactory;
+import { TDetectoryFactoryOptions, THandler } from 'element-resize-detector';
+if (typeof __IS_NODE__ === 'undefined') {
```

Comment:
what's __IS_NODE__, how does it relate to components lib?
let's check smth like `typeof document === 'undefined' or `typeof window === 'undefined'`